src/main/java/org/osiam/client/OsiamConnector.java

```diff
@@ -65,7 +69,8 @@
     private static final int DEFAULT_MAX_CONNECTIONS_PER_ROUTE = 20;
 
     private static final PoolingHttpClientConnectionManager CONNECTION_MANAGER;
-    private static final Client CLIENT;
+    private static final javax.ws.rs.client.Client CLIENT;

```

Comment:
This is not a traditional constant. Please don't name it all upper case. (We agreed on something similar with the loggers, remember?)